### ISSUE TITLE
Having black screen problems with Japanese Pokemon Colosseum Celebi Disk

### ISSUE THREAD

**Issue by astronomicay** [FIX94/Nintendont#1225]
Hello everyone.
I'm trying to transfer the special celebi from the disk to pokemon colosseum (using an original disk for celebi and a save file downloaded online even though I have a Japanese copy of Pokemon Colosseum (running on a pal console through NINTENDONT).
I'm experiencing the same problem as #195 mentioned (and another person as well, I don't remember the code sorry). I really don't have any idea how to get it work, the pokemon channel video loads with no problem but whenever I try to go on with the celebi transfer the screen goes black and don't load anything.

**Comment by astronomicay** [comment]
The other person is #1054

**Comment by carnage702** [comment]
> Hello everyone. I'm trying to transfer the special celebi from the disk to pokemon colosseum (using an original disk for celebi and a save file downloaded online even though I have a Japanese copy of Pokemon Colosseum (running on a pal console through NINTENDONT). I'm experiencing the same problem as #195 mentioned (and another person as well, I don't remember the code sorry). I really don't have any idea how to get it work, the pokemon channel video loads with no problem but whenever I try to go on with the celebi transfer the screen goes black and don't load anything.

first off pokemon channel does not have any celebi to transfer

jap has a colosseum bonus disk that can transfer celebi, pokemon channel japan has no transfers

Pokemon channel PAL and NTSC has a jirachi to transfer.


if you want a celebi you need a japanese version of ruby saphire on the gba  and using the gba to gc  cable using the colosseum bonus disk both games need to have same languague if you are using a ruby or saphire that inst japanese it wont work.

**Comment by astronomicay** [comment]
Yeah, I know about it. I actually bought a Japanese celebi disk (a version that includes a promotional version of Pokemon channel) and I have a Japanese copy of ruby as well to transfer celebi from colosseum to the gba. My issue is not being able to go further into the celebi disk because, after selecting the celebi option, the screen turn black and I'm unable to do anything else.

**Comment by carnage702** [comment]
what are your nintendont settings post them here a screenshot would be better

**Comment by astronomicay** [comment]
![image](https://github.com/user-attachments/assets/f86ef868-ccac-4549-99de-be6e08ab424a)
![image](https://github.com/user-attachments/assets/ab829833-9f2b-485d-8531-e4b43f54499a)
![image](https://github.com/user-attachments/assets/03944ab1-225d-4667-8fb5-2294b5833ff2)
![image](https://github.com/user-attachments/assets/41160a31-6329-4ba5-a6bb-7325f0715d15)
![image](https://github.com/user-attachments/assets/f3a03127-2d16-4c75-9650-638547f43ffb)
![image](https://github.com/user-attachments/assets/75b5dd16-3edf-4fe1-9b0e-3b9f3bd3403d)
![image](https://github.com/user-attachments/assets/55f8af2e-63df-4b8a-8efa-dd813467979a)
![image](https://github.com/user-attachments/assets/3a00aeda-8bcf-4cd9-af8b-a8155d250be5)

I tried this settings because I thought they were pretty similar to the one that issues #195 and #1048 used to fix their problems. I heard that for the celebi disk there are 2 versions, maybe mine is still to be patched?

**Comment by carnage702** [comment]
first off dont use nintendont through usbloadergx or anything, just use nintendont directly from homebrew channel that way its easier to make sure the settings are correct and set.

second you are using a real memory card? that could be an issue too.

so my advice is make a multimemory japanese card which will be called something ninmemj.raw  inject your colosseum save file inside that memory card which you can use dolphin to do so.

you are using a regular wii with gc ports right? 

make sure every nintendont setting is off or auto with exception being memory card emulation set to on multi set to on and native set to on(needed to read the gba link cable from wii with gc ports) and make sure your multi memory card jap is on the saves folder

**Comment by astronomicay** [comment]
![image](https://github.com/user-attachments/assets/b60a1fba-68de-4973-af7f-0bc1a5a12f1f)
![image](https://github.com/user-attachments/assets/deb095d4-6931-4ecf-965e-e1ef95b81406)
![image](https://github.com/user-attachments/assets/f4bcd1d3-d1d8-4101-8b6d-c2932ab87f38)
![image](https://github.com/user-attachments/assets/84c1344b-733c-435c-905a-1abf998a0b27)

I tried it but it seems to not work. Are some of my settings wrong? Should I try to create a new ninmemj.raw memory card?

**Comment by carnage702** [comment]
how did you created that one? did you let nintendont creat it? or just rename some? also are you sure your cd is good?

**Comment by carnage702** [comment]
did you removed your memory card from the slot?

**Comment by astronomicay** [comment]
Yeah, The .raw file was created from nintendont through usbloadergx. I also took off the memory card from the console. I bought the disk from eBay and the seller said that "operation have been confirmed"

**Comment by carnage702** [comment]
also if you have  everything couldnt you just use regular gc mode instead of nintendont like dios mios? real cd real memory card and real controllers should work

**Comment by astronomicay** [comment]
I play on  PAL Wii and if I try to boot up both the Japanese colosseum disk and the bonus disk I only get a black screen (but I can play pal pokemon colosseum with no issues)

**Comment by carnage702** [comment]
> I play on PAL Wii and if I try to boot up both the Japanese colosseum disk and the bonus disk I only get a black screen (but I can play pal pokemon colosseum with no issues)

dios mios is region free as far as i know

**Comment by astronomicay** [comment]
Oh so you're suggesting to use an alternative gc games player?

**Comment by carnage702** [comment]
> Oh so you're suggesting to use an alternative gc games player?

to do what you want its an alternative i guess, i dont have the hardware to test  if its your error or nintendont since ppl already did it im guessing something must be off not sure what maybe try getting a copy of the bonus disc on sd/usb and try that as last resort? i cant test this since i dont own jap ruby or saphire i did try jirachi from pokemon channel and works so its the best i can try.

**Comment by SLIMEMASTER000** [comment]
im having this same exact issue... exept im not using a real disc but an ISO file in the games folder, ether way nomater what i try all i get is a black screen when i try to press A at the celebi option /:
